pandas/tests/groupby/test_groupby.py

```diff
@@ -199,7 +199,7 @@ def f_1(grp):
     with tm.assert_produces_warning(FutureWarning, match=msg):
         result = df.groupby("A").apply(f_1)[["B"]]
     e = expected.copy()
-    e.loc["Tiger"] = np.nan
+    e["B"] = [3, 5, float("nan")]
```

Comment:
why is this changing?  any reason for `float('nan')` instead of np.nan?

Reply:
indeed, it didn't need to change, thanks